docs/js/global-trap-stack.js

```diff
@@ -0,0 +1,26 @@
+const { createFocusTrap } = require('../../index');
+
+module.exports = () => {
+  const container = document.getElementById('global-trap-stack');
+  const counter = container.querySelector('.counter');
+  const trapStack = []
```

Comment:
Could we stick this in the `window` and then add 2 checks in the related test:
1. After the trap is activated, there's an entry in the array.
2. After the trap is deactivated, the array is empty.

That would make the test more meaningful. Right now, it's just a duplicate of the default example test.